_posts/2023-03-03-broadband-fabric-data-challenge.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "The Broadband Fabric Data Challenge Process: A Tale of Collaboration"
+author:
+  display_name: Jake Adams
+  email: jdadams@utah.gov
+date: 2023-03-01 12:00:00
+categories:
+  - Featured
+tags:
+  - data
+  - broadband
+---
+
+_From the Teamwork Makes the Dream Work department_
+{: .flex .flex--center}
+
+Among the many lessons the Covid-19 pandemic taught us, one that was hammered home almost immediately was the importance of high-speed broadband internet access at home. As work-from-home and simultaneous video calls became commonplace, we discovered that the asymmetrical, download-focused broadband of yesteryear just didn't cut it anymore. We also discovered that figuring out what broadband was available in our neighborhood was a more difficult task than it had to be.
+
+Unfortunately, this isn't new—the state and federal governments have been struggling with poor data on broadband availability for years. While the FCC required providers to report coverage, the reliance on census block geometries made it impossible to get an accurate and fine-grained picture. Without accurate data, governments have struggled to direct resources and policy to areas that are lacking coverage or the competition necessary for free market economics to do their thing. Ultimately, consumers have often been the ones to suffer slow speeds and high prices.
```

Comment:
**thought**: these fancy dash options are available to you if you so please.
<img width="599" alt="image" src="https://user-images.githubusercontent.com/325813/222517376-0dbd7459-df98-4015-b2a0-47c11ff597a5.png">